src/tools/elektra-gpg-testkey/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+find_package (LibGpgme QUIET)
+if (NOT LIBGPGME_FOUND)
+    remove_tool (elektra-gpg-testkey "libgpgme not found")
+else ()
+    include_directories (${LIBGPGME_INCLUDE_DIRS})
+    add_executable (elektra-gpg-testkey elektra-gpg-testkey.c)
+    target_link_libraries (elektra-gpg-testkey ${LIBGPGME_LIBRARIES})
+    install (TARGETS elektra-gpg-testkey DESTINATION bin)
```

Comment:
What about installing it to TARGET_TOOL_EXEC_FOLDER?

Then you can also call it gen-gpg-testkey and call it via `kdb gen-gpg-testkey`.

Reply:
Good idea!